### PR TITLE
Fixed '[WARNING] There were 0 download errors - check the warnings above to not log warning unless download errors > 0

### PR DESCRIPTION
**Pull request by blekinge**
Really not much to say here. It is just annoying to see the warning in the maven log when where really is nothing to warn about

### CODE REVIEW

File: src/main/java/org/codehaus/mojo/license/AbstractDownloadLicensesMojo.java [hunk 1]
```diff
@@ -822,25 +822,20 @@ public void execute()
...
-                if ( downloadErrorCount > 0 )
-                {
-                    throw new MojoFailureException( "There were " + downloadErrorCount + " download errors - check "
-                        + licensesErrorsFile.getAbsolutePath() );
-                }
-                break;
-            default:
-                throw new IllegalStateException( "Unexpected value of " + ErrorRemedy.class.getName() + ": "
-                    + errorRemedy );
+        if ( downloadErrorCount > 0 ) {
```
Comment (nfalco79) [inline src/main/java/org/codehaus/mojo/license/AbstractDownloadLicensesMojo.java:825]: double check formatting

**Comment by george-kay** [comment]
I can't find the change in latest sources. Could someone approve it?

**Comment by nfalco79** [comment]
I'm not the owner of this plugin so my approval count 0

**Review by nfalco79 — APPROVED** [review]

**Comment by george-kay** [comment]
@ppalaga could you approve it? It bugs our automation with this error message. Or just tell us what todo. Thx a bunch!

**Comment by FlorianWolters** [comment]
Can this be merged please? The warning is still occuring with v2.1.0:

    [INFO] --- license:2.1.0:download-licenses (download-licenses) @ hello_world ---
    [WARNING] There were 0 download errors - check the warnings above

**Pull request merged**